azure-monitor-opentelemetry/README.md

```diff
@@ -53,11 +53,19 @@ pip install azure-monitor-opentelemetry --pre
 You can use `configure_azure_monitor` to set up instrumentation for your app to Azure Monitor. `configure_azure_monitor` supports the following optional arguments:
 
 * connection_string - The [connection string][connection_string_doc] for your Application Insights resource. The connection string will be automatically populated from the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable if not explicitly passed in.
-* disable_logging - If set to `True`, disables collection and export of logging telemetry. Defaults to `False`.
-* disable_metrics - If set to `True`, disables collection and export of metric telemetry. Defaults to `False`.
-* disable_tracing - If set to `True`, disables collection and export of distributed tracing telemetry. Defaults to `False`.
-* exclude_instrumentations - By default, all supported [instrumentations](#officially-supported-instrumentations) are enabled to collect telemetry. Specify instrumentations you do not want to enable to collect telemetry by passing in a comma separated list of instrumented library names. e.g. `["requests", "flask"]`
-* instrumentation_config - Specifies a dictionary of kwargs that will be applied to instrumentation configuration. You can specify which instrumentation you want to configure by name in the key field and value as a dictionary representing `kwargs` for the corresponding instrumentation.
+* exclude_instrumentations - By default, all supported [instrumentations](#officially-supported-instrumentations) are enabled to collect telemetry. Specify instrumentations you do not want to enable to collect telemetry by passing in a comma separated list of instrumented library names. Can also be set via the `APPLICATIONINSIGHTS_EXCLUDE_INSTRUMENTATIONS` environment variable. e.g. `["requests", "flask"]`
```

Comment:
Have the `APPLICATION_INSIGHTS_*` env var configurations been discussed with the other distro owners?